docker-compose.yml

```diff
@@ -49,11 +49,15 @@ services:
     networks:
       elonet:
         ipv4_address: 172.16.11.5
+  elo_fe:
+    container_name: elo_fe
+    image: node:10
+    user: "node"
+    working_dir: /home/node/app
+    volumes:
+      - $SOURCE_DIR/fe:/home/node/app
+    tty: true  
 networks:
   elonet:
     driver: bridge
-    ipam:
```

Comment:
Tutaj niewiem dokońca do czego to służy. Jak wywaliłem apka działała, więc niewie czy to potrzebne ?

Reply:
Tak, to jest raczej potrzebne, ponieważ wymusza od Dockera strukturę danej sieci kontenerów w obrębie tego projektu. Jeśli tego nie będzie, to użyje domyślnych, które mogą wchodzić w konflikt z innymi włączonymi kontenerami, a nawet i całą siecią na kompie. Choć kwestia czy użyje, bądź nie, tych adresów zawartych w "ipv4_address".

Swoją drogą, dla kontenera "elo_fe" musiałbyś określić jeszcze właśnie wartość "ipv4_address".

Reply:
@ktrzos 
Usunąłem to głwnie dlatego bo w wersji 3 docker-compse nie wspiera tego ipam. No ale spoko zmieniłem wersję spowrotem na 2 i przywróciłem to. Oprócz tego nadałem ipv4 dla elo_fe

Reply:
Ooo, to jest jakaś 3. wersja :D? Trza by to wybadać. Można dodać kejsa aby przerobić tą wersję drugą na trzecią z odpowiednim zachowaniem konfiguracji.

Reply:
@ktrzos 
No jest trzecia wersja: https://docs.docker.com/compose/compose-file/
Założyłem issuesa jak coś.